### DISCUSSION TITLE
error message: Templating [car_id] Error updating options

### DISCUSSION THREAD

**Discussion by kikupekr** [teslamate-org/teslamate#2321] [Q&A] [answered]
1)teslamate stopped getting info from my car about 2 days ago(afternoon of jan 12, 2022)
(i can see the graphs on the mileage and charge level dashboards. they ends on jan 12, 2022)

2)updated teslamate to 1.25.2. logged in to grafana. cannot connect to the port 4000.

3)on grafana, i can see the dashboards of teslamate. but these do not yields any data.
keep getting red error message from many of the dashboards
"Templating [car_id]
Error updating options: pq: column "display_priority" does not exist"

4)teslamate seems to keep restarting.
teslamate/teslamate:latest   "tini -- /bin/sh /en…"   9 hours ago   Restarting (134) 35 seconds ago

5)

**Comment by evilmouse69** [comment]
I have the same issue

root@teslamate-i:~# docker ps -a
CONTAINER ID        IMAGE                        COMMAND                  CREATED             STATUS                            PORTS                    NAMES
c330b702215d        eclipse-mosquitto:1.6        "/docker-entrypoint.…"   7 minutes ago       Up About a minute                 0.0.0.0:1883->1883/tcp   default_mosquitto_1
ceb648d550e4        teslamate/teslamate:latest   "tini -- /bin/sh /en…"   7 minutes ago       Restarting (134) 10 seconds ago                            default_teslamate_1
dfd15ad4ab7b        postgres:13                  "docker-entrypoint.s…"   7 minutes ago       Up About a minute                 5432/tcp                 default_database_1
b106150f0999        teslamate/grafana:latest     "/run.sh"                7 minutes ago       Up About a minute                 0.0.0.0:3000->3000/tcp   default_grafana_1

**Comment by adriankumpf** [comment]
Can you post the complete logs right after a restart of the app? It looks like the database migrations, which should run automatically when the Docker container starts, did not run.

**Comment by rameshputrevu** [comment]
Hello Adrian, 

I have the similar issue.. Did you get the logs from anyone? If not please let me know how and where to get the logs and I will send those..

Thank you
Ramesh

**Reply by cwanja** [reply]
To retrieve the TeslaMate Docker container logs, run `docker-compose logs teslamate`.

**Reply by rameshputrevu** [reply]
from the log files, I see these 2 lines repeatedly..

teslamate_1  | clock_gettime(CLOCK_MONOTONIC, _) failed: Operation not permitted (1)
teslamate_1  | Aborted (core dumped)

**Reply by cwanja** [reply]
Are you running a Raspberry Pi? I assume you are running a Raspberry Pi on Buster? You would need to apply the fixes per #2302. Or upgrade your Raspberry Pi to Bullseye. See the release notes in 1.26 - https://github.com/adriankumpf/teslamate/releases/tag/v1.26.0 https://github.com/adriankumpf/teslamate/discussions/2321#discussioncomment-2068668

**Comment by cwanja** [comment, marked as answer]
With regards to point 1, I wonder if you fell victim to the API token being revoked. I think your timeline was around the time that Tesla pulled a handful of API tokens due to a security issue with exposed TeslaMate installations to the internet. Read more here. https://medium.com/@david_colombo/how-i-got-access-to-25-teslas-around-the-world-by-accident-and-curiosity-8b9ef040a028

For point 2, I assume you are running a Raspberry Pi on Buster? You would need to apply the fixes per #2302. Or upgrade your Raspberry Pi to Bullseye. See the release notes in 1.26 - https://github.com/adriankumpf/teslamate/releases/tag/v1.26.0

> Bump Docker app base image to Debian 11
> Raspberry Pi users unfortunately have to upgrade to Raspbian Bullseye or install the backports version libseccomp2 (see #2302)

**Reply by evilmouse69** [reply]
Both of these fixed my issue. Upgrade from buster to bullseye and then re-auth for Tesla
